Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt

```diff
@@ -128,6 +147,22 @@ class DartToolingDaemonService private constructor(private val project: Project)
     webSocket.send(requestString)
   }
 
+  fun sendResponse(id: String, result: JsonObject) {
```

Comment:
I made this function private. It is ok?

Reply:
Yes it should only be used in this class